docks.py

```diff
@@ -0,0 +1,878 @@
+                               QMessageBox, QToolButton, QColorDialog,
+                               QDockWidget, QItemDelegate, QHeaderView,
+                               QSlider, QScrollArea, QTextEdit, QListWidget,
+                               QListWidgetItem, QTreeWidget, QTreeWidgetItem)
+
+from matplotlib import cm as mcolormaps
+import numpy as np
+
+from openmc.filter import (UniverseFilter, MaterialFilter, CellFilter,
+                           SurfaceFilter, MeshFilter, MeshSurfaceFilter)
+
+from custom_widgets import HorizontalLine, Expander
+from scientific_spin_box import ScientificDoubleSpinBox
+
+_SPATIAL_FILTERS = (UniverseFilter, MaterialFilter, CellFilter,
+                    SurfaceFilter, MeshFilter, MeshSurfaceFilter)
+
+reaction_units = 'Reactions per Source Particle'
+flux_units = 'Particle-cm per Source Particle'
+production_units = 'Particles Produced per Source Particle'
+energy_units = 'eV per Source Particle'
+
+reactions = ('absorption', 'elastic', 'fission',
+             'scatter', 'total', '(n,2nd)',
+             '(n,2n)', '(n,3n)', '(n,na)',
+             '(n,n3a)', '(n,2na)', '(n,np)',
+             '(n,n2a)', '(n,2n2a)', '(n,nd)',
+             '(n,nt)', '(n,nHe-3)', '(n,nd2a)',
+             '(n,nt2a)', '(n,n4n)', '(n,2np)',
+             '(n,3np)', '(n,n2p)', '(n,n*X*)',
+             '(n,nc)', '(n,gamma)', '(n,elastic)',
+             '(n,p)', '(n,d)', '(n,t)',
+             '(n,3He)', '(n,a)', '(n,2a)',
+             '(n,3a)', '(n,2p)', '(n,pa)',
+             '(n,t2a)', '(n,d2a)', '(n,pd)',
+             '(n,pt)', '(n,da)')
+productions = ('delayed-nu-fission', 'prompt-nu-fission', 'nu-fission',
+               'nu-scatter', 'H1-production', 'H2-production',
+               'H3-production', 'He3-production', 'He4-production')
+
+score_units = {}
+score_units.update({reaction : reaction_units for reaction in reactions})
+score_units.update({production : production_units for production in productions})
+score_units['flux'] = 'Particle-cm/Particle'
+score_units['current'] = 'Particles per source Particle'
+score_units['events'] = 'Events per Source Particle'
+score_units['inverse-velocity'] = 'Particle-seconds per Source Particle'
+score_units['heating'] = energy_units
+score_units['heating-local'] = energy_units
+score_units['kappa-fission'] = energy_units
+score_units['fission-q-prompt'] = energy_units
+score_units['fission-q-recoverable'] = energy_units
+score_units['decay-rate'] = 'Seconds^-1'
+score_units['damage-energy'] = energy_units
+
+tally_values = {'Mean': 'mean',
+                'Std. Dev.': 'std_dev',
+                'Rel. Error': 'rel_err'}
+
+class PlotterDock(QDockWidget):
+    """
+    Dock widget with common settings for the plotting application
+    """
+    def __init__(self, model, FM, parent=None):
+        super().__init__(parent)
+
+        self.model = model
+        self.FM = FM
+        self.mw = parent
```

Comment:
Yes, otherwise the reader may not know what these are. I can guess that mw = main window, but I honestly don't know what "FM" is or why it's capitalized (and mw is not).

Reply:
Fair enough! Updated those variable names throughout.